.github/workflows/renovate.yaml

```diff
@@ -0,0 +1,41 @@
+name: Renovate
+on:
+  # running every two hours from 9h to 19h UTC on every working day
+  schedule:
+    - cron: '0 9-19/2 * * 1-5'
+  # allow to manually trigger this workflow
+  workflow_dispatch:
+    inputs:
+      renovate_log_level_debug:
+        type: boolean
+        default: true
```

Comment:
@meyskens I've been thinking a little bit more about the "scheduled" which is why it took me more time to review this PR.

Can't we do every hour between 9-19 and then 2 hours outside of it including weekends **as well as** run it every time a pull request is merged into main?

Reply:
Sure we can, this is what was used on tetragon so i copied the schedule that was already working for them. Will add more :)

Reply:
After a close consideration i have the counter offer of every two hours in weekends and every hour on weekdays since we have active Cilium contributors in every continent focusing on UTC workhours might be not the best idea.